geonode/assets/utils.py

```diff
@@ -98,27 +127,32 @@ def create_asset_and_link(
     clone_files: bool = True,
 ) -> tuple[Asset, Link]:
 
-    asset_handler = handler or asset_handler_registry.get_default_handler()
     asset = link = None
     try:
-        default_title, default_ext = os.path.splitext(next(f for f in files)) if len(files) else (None, None)
+        first_file = next(iter(files)) if len(files) else None
+        filename = (
+            first_file
+            if isinstance(first_file, (str, Path))
+            else getattr(first_file, "name", None) if first_file else None
+        )
+        default_title, default_ext = os.path.splitext(filename) if filename else (None, None)
```

Comment:
@sijandh35 why do you set the `default_title` to `None` in case we cannot get the title from the `filename`, then you make a triple or in `create_asset` and set to "Unkknow" if default_title is `None`?
Can't you just set `default_title` to "Unknonw" here?